os2phonebook_service/os2phonebook/controller.py

```diff
@@ -250,6 +511,8 @@ def call_search_method():
 @api.app_errorhandler(NotFoundError)
 @api.app_errorhandler(InvalidSearchType)
 @api.app_errorhandler(InvalidRequestBody)
+@api.app_errorhandler(InvalidCredentials)
+@api.app_errorhandler(InsufficientCredentials)
 def invalid_validation_handler(error) -> Response:
     """Error handler for all common types
 
```

Comment:
Update the docstring below with the new errors (or maybe just remove the bit detailing the error types - it seems a bit redundant)

Reply:
See 18fe897